src/CurrencyTransformer/AzerbaijaniCurrencyTransformer.php

```diff
@@ -37,7 +37,7 @@ public function toWords(int $amount, string $currency, ?CurrencyTransformerOptio
 
         if (!array_key_exists($currency, AzerbaijaniDictionary::$currencyNames)) {
             throw new NumberToWordsException(
-                sprintf('Currency "%s" is not available for "%s" language', $currency, get_class($this))
+                sprintf('"%s" valyutası "%s" dilində mövcud deyil', $currency, get_class($this))
```

Comment:
The exceptions and generic messages should be all in English. Could you please revert it?